pos/modules/checkout/hooks/useCapitron.tsx

```diff
@@ -27,7 +27,7 @@ const useCapitron = () => {
 }
 
 export const useCapitronTransaction = (options: {
-  onCompleted: () => void
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider specifying a more detailed type than `any` for `onCompleted` parameter.

Using `any` can lead to potential type safety issues. If possible, define a more specific type that reflects the structure of the expected parameter.

```suggestion
  onCompleted: (info?: { [key: string]: any }) => void
```